index.xml

```diff
@@ -1428,6 +1428,14 @@
         <cd:MatchData offset="95">SD-DoRIT-L1B-006 v4.15 </cd:MatchData>
       </cd:DetectionRule>
     </cd:ProductDefinition>
+    <cd:ProductDefinition id="ALD_U_N_1B_04_16" format="binary" version="12">
+      <cd:Description>This definition is based on version 4/16 of IODD AED-SD-DoRIT-L1B-006</cd:Description>
+      <cd:DetectionRule>
+        <cd:MatchData offset="0">PRODUCT="AE_</cd:MatchData>
+        <cd:MatchData offset="17">ALD_U_N_1B</cd:MatchData>
+        <cd:MatchData offset="95">SD-DoRIT-L1B-006 v4.16 </cd:MatchData>
+      </cd:DetectionRule>
+    </cd:ProductDefinition>
   </cd:ProductType>
   <cd:ProductType name="ALD_U_N_1B_HDR">
     <cd:Description>Aeolus Level 1B Wind Measurement Product (HDR)</cd:Description>
```

Comment:
I saw this too I think.. if coda does not recognize the product (product_class=product_type=None,version=-1), codacheck  just says all good!?

Reply:
In order for codacheck to raise an error if no codadef definition could be found, make sure to pass the '-d' option.
Otherwise it will just happily handle self-describing product formats without complaining.